app/decorators/ext_management_system_decorator.rb

```diff
@@ -6,6 +6,10 @@ def fonticon
   end
 
   def listicon_image
-    "100/vendor-#{image_name}.png"
+    if kind_of?(ManageIQ::Providers::Hawkular::MiddlewareManager)
+      'vendor-hawkular'
+    else
+      "100/vendor-#{image_name}.png"
```

Comment:
Wait, isn't `ManageIQ::Providers::Hawkular::MiddlewareManager` a subclass of `ExtManagementSystem`? You should create `app/decorators/manageiq/providers/hawkular/middleware_manager_decorator.rb` with `def listicon_image; 'vendor-hawkular'; end` instead of hardcoding the if here...